src/classes/CW_CampaignSitesTriggerHandlerTest.cls

```diff
@@ -0,0 +1,68 @@
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.